documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -23,66 +23,78 @@ Namada groups public goods into four categories, with earmarked pools of funding
 - Social research, art, and philosophy
   _Social research, art, and philosophy_ covers funding for artistic expression, philosophical investigation, and social/community research (_not_ marketing) exploring the relationship between humans and technology. Possible funding forms could include independent artist grants, institutional funding, funding for specific research resources (e.g. travel expenses to a location to conduct a case study), and similar.
+- Education
+  _Education_ covers the funding for open and free to use knowledge, compiled and/or produced by educators in various forms. This can include authors of books, blog-posts, podcasts websites and other educational materials. In a sense, this is a type of *meta public good*, as open knowledge often sparks more open knowledge, although not necessarily
+
+- Meta Public Goods
+_Meta public goods_ covers funding for any good that increases the production or existence of other public goods. The management of forums, libraries, quadratic funding protocols, dominant assurance contracts, etc. are good examples of this.
+
 - External public goods
   _External public goods_ covers funding for public goods explicitly external to the Namada and Namada ecosystem, including carbon sequestration, independent journalism, direct cash transfers, legal advocacy, etc. Possible funding forms could include direct purchase of tokenised assets such as carbon credits, direct cash transfers (e.g. GiveDirectly), institutional funding (e.g. Wikileaks), and similar.
 
-### Funding amounts
+## The Public Goods Stewards
 
-In Namada, up to 10% inflation per annum of the NAM token is directed to this public goods mechanism. The further division of these funds is entirely up to the discretion of the elected PGF council.
+The funding of public goods on Namada will be conducted through a structure we call "Public Goods Stewards".
 
-Namada encourages the public goods council to adopt a default social consensus of an equal split between categories, meaning 1.25% per annum inflation for each category (e.g. 1.25% for technical research continuous funding, 1.25% for technical research retroactive PGF). If no qualified recipients are available, funds may be redirected or burnt.
+Stewards are elected by governance through a governance proposal. Each Steward will be specialised in funding a subset of categories, which they specify during their election. Stewards can then "propose" funding of various public goods, which will pass by default. However, Governance retains the power to veto any proposal, which would result in the Steward being removed from the set of Stewards.
```

Comment:
I'd say - each steward will be responsible for covering a specific area of public goods, which they describe during their election.

The areas don't need to exactly correspond to the categories - actually, we should probably get rid of "categories" and just list a bunch of different kinds of public goods which Namada is interested in funding.